.github/workflows/gradle.yml

```diff
@@ -729,8 +736,9 @@ jobs:
             --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED \
             --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED \
             --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED
-          org.gradle.workers.max=8
+          org.gradle.workers.max=512
```

Comment:
@xpuska513 did we need this for tests to work?

Reply:
nah, I can revert it back to the default value, but I saw that the same value was set in kube AC tests so did the same here.